sql/catalyst/src/main/scala-2.12/org/apache/spark/sql/catalyst/expressions/AttributeMap.scala

```diff
@@ -50,4 +50,6 @@ class AttributeMap[A](val baseMap: Map[ExprId, (Attribute, A)])
   override def iterator: Iterator[(Attribute, A)] = baseMap.valuesIterator
 
   override def -(key: Attribute): Map[Attribute, A] = baseMap.values.toMap - key
+
+  def ++(other: AttributeMap[A]): AttributeMap[A] = new AttributeMap(baseMap ++ other.baseMap)
```

Comment:
This is fixing `src/main/scala-2.12` only. We need to do the same thing for `src/main/scala-2.13`.